src/foram3d/simulation.ts

```diff
@@ -239,6 +251,11 @@ module Foram3D {
       ]
     }
 
+    private setupMouseEvents() {
+      window.addEventListener('click',     (event: Event) => this.onMouseClick(event));
```

Comment:
Również pod względem wydajności, lepiej byłoby bindować się do `canvas` niż do `window`, ale musisz sprawdzić jak jest z kompatybilnością bindowania MouseEvents na elementy. IMO powinno działać (na Chrome działa :P) ale jeżeli wsparcie jest słabe to nie da rady tego tak zrobić.